### PR TITLE
Issue-9949 Helper icon for prompt component

### LINKED ISSUE #9949: [Extension][Cloud] - Helper icon for prompt component
### Current Behavior

Screenshot 2024-01-13 at 9 02 35 PM

### Desired Behavior

- Add helper icon for the prompt component.
- When hover over the icon tooltip message have to appear.

Screenshot 2024-01-13 at 8 56 24 PM

Screenshot 2024-01-13 at 9 00 28 PM

### Mockups

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](http://discuss.meshery.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by anmolchhabra21**
**Notes for Reviewers**

This PR fixes #9949 

In this PR, it was required to add a right-side Helper prompt icon. I have built it in this way so that we can send a message named `helpText`, and it will automatically add it to the helper prompt. Also, I have made a field of adding something in the future in the left-side heading also, currently, it is just showing blank spaces. 
Currently, on right-side, it displays the text `Deletes the "${environment.name}" Environment`, as in the following picture.

image

Also, In the files where we don't require the left nor right prompt, It will just render the text, as it was previously.

Meshery_2

Please let me know if any improvements can be made... @senali-d @leecalcote 

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [-] Yes, I signed my commits.
 Signed-off-by: Anmol Chhabra <anmolchhabra71500@gmail.com>

### CODE REVIEW

File: ui/components/Lifecycle/Environments/index.js [hunk 1]
```diff
@@ -296,6 +296,7 @@ const Environments = ({ organization, classes }) => {
       
       Learn more about the behavior of [lifecycle of environments and their resources](https://docs.meshery.io/concepts/logical/environments) in Meshery Docs.`,
       variant: PROMPT_VARIANTS.DANGER,
+      helpText: `Deletes the "${environment.name}" Environment`,
```
Comment (sudhanshutech) [inline ui/components/Lifecycle/Environments/index.js:299]: not required, the environment name here, its been already shown inside modal

File: ui/components/PromptComponent.js [hunk 2]
```diff
@@ -27,6 +28,24 @@ const styles = (theme) => ({
     backgroundColor:
       theme.palette.type === 'dark' ? theme.palette.secondary.headerColor : '#396679',
```
Comment (sudhanshutech) [inline ui/components/PromptComponent.js:29]: use theme reference, no hex

File: ui/components/PromptComponent.js [hunk 3]
```diff
@@ -27,6 +28,24 @@ const styles = (theme) => ({
...
+    padding: '10px',
+    color: '#fff',
+    '& h2': {
+      display: 'flex',
+      justifyContent: 'space-between',
+      alignItems: 'center',
+      textAlign: 'center',
+    },
+    backgroundColor:
+      theme.palette.type === 'dark' ? theme.palette.secondary.headerColor : '#396679',
```
Comment (sudhanshutech) [inline ui/components/PromptComponent.js:43]: use theme reference, no hex

File: ui/components/PromptComponent.js [hunk 4]
```diff
@@ -27,6 +28,24 @@ const styles = (theme) => ({
     backgroundColor:
       theme.palette.type === 'dark' ? theme.palette.secondary.headerColor : '#396679',
   },
+  titleHeading: {
+    textAlign: 'center',
+    minWidth: 400,
+    padding: '10px',
+    color: '#fff',
```
Comment (sudhanshutech) [inline ui/components/PromptComponent.js:35]: use theme reference

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Review by sudhanshutech — COMMENTED** [review]
@anmolchhabra21 have you confirmed the helper icon works same for workspaces or the places this prompt is rendered?

**Comment by sudhanshutech** [comment]
@anmolchhabra21 some checks are failing.
Also please it is a good practice to get assigned to the issue first, would you comment on the issue so i can assign you @anmolchhabra21

File: ui/components/Lifecycle/Environments/index.js (hunk 1 above)
Comment (anmolchhabra21) [reply ui/components/Lifecycle/Environments/index.js:299]: Sure, will remove the variable, and helper prompt will display **Deletes the Environment**, would that be okay?

**Comment by leecalcote** [comment]
@anmolchhabra21 light and dark mode are both displaying the correct color?

**Comment by anmolchhabra21** [comment]
Yes @leecalcote, it is displaying correct colors as per the theme now. 

image
image

**Comment by anmolchhabra21** [comment]
Hi @leecalcote, @sudhanshutech, I am not sure why the tests are failing, I made a empty commit, to just run it once again, but it passed the previously failed and now failing on another test, can you please re-run them, 
I have re-checked my commits, and they are not associated with my changes, I think these are randomly failing, can you please help me with it.

**Comment by leecalcote** [comment]
Please add a link to the Environments docs page.

**Comment by anmolchhabra21** [comment]
@sudhanshutech, I think the PR is ready for review. Can you please have a look at it once you are done with your tasks? 
Regarding the failing snapshot, I asked for it in the Slack channel, and It can be ignored.

**Comment by anmolchhabra21** [comment]
Any update for me on this PR?

**Pull request closed without merging**